build/build.xml

```diff
@@ -17,6 +17,7 @@ External Dependencies:
  	<property name="distro.name"				value="commandbox"/>
  	<property name="commandbox.version"			value="3.0.1"/>
  	<property name="commandbox.stableVersion"	value="3.0.0"/>
+ 	<property name="commandbox.stableSHA256"	value="194738f683cbbbaa63f7a2ae5d1fe6c0d09ef8370c4ac0b6c9aa087f6013abc1" />

```

Comment:
I wasn't aware there was a physical file that maintained that hash.  Pulling the hash from that file works too.

Having that hash for stable during the build is necessary to ensure that our brew file is always current with the stable and dev versions and hashes.  Our current build process basically creates a "disposable" commandbox.rb that is only ever used with a major release.

> On Feb 21, 2016, at 12:39 AM, Brad Wood <notifications@github.com> wrote:
> 
> In build/build.xml:
> 
> > @@ -17,6 +17,7 @@ External Dependencies:
> >   	<property name="distro.name"				value="commandbox"/>
> >   	<property name="commandbox.version"			value="3.0.1"/>
> >   	<property name="commandbox.stableVersion"	value="3.0.0"/>
> > + 	<property name="commandbox.stableSHA256"	value="194738f683cbbbaa63f7a2ae5d1fe6c0d09ef8370c4ac0b6c9aa087f6013abc1" />
> How does this hash get update? I'm not quite following why we need it. Once we make the final build for a stable version, the hash will never change, so the brew file that's placed in that directory should always have the correct matching hash, right? For instance, the current stable brew file here http://integration.stg.ortussolutions.com/artifacts/ortussolutions/commandbox/3.0.0/commandbox.rb will never be updated again and it has the matching hash that was generated the final time we ran that build. Why do we need it in the source as well? if you're wanting to read it in a later build, shouldn't we be storing it in the same manner that we store the other hashes?
> http://integration.stg.ortussolutions.com/artifacts/ortussolutions/commandbox/3.0.0/commandbox-bin-3.0.0.zip.sha
> 
> —
> Reply to this email directly or view it on GitHub.
>